frontend/src/assets/i18n/it.json

```
@@ -357,5 +357,755 @@
+        },
+        "nav": {
+            "howItWorks": "Come funziona",
+            "advantages": "Vantaggi",
+            "features": "Caratteristiche",
+            "pricing": "Prezzi",
+            "selfHost": "Self-hosting",
+            "faq": "FAQ"
+        },
+        "actions": {
+            "signIn": "Accedi",
+            "claimSpot": "Rivendica il tuo posto",
```

Biome (2.3.13)

[error] The key actions was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)